charts/telegraf-ds/values.yaml

```diff
@@ -81,3 +81,5 @@ config:
         password: ""
         user_agent: "telegraf"
         insecure_skip_verify: false
+  inputs:
```

Comment:
I'd like not to give any sort of indication that input configuration is available in this chart, so I'd like to suggest we rename this option. 

Current thought is `monitor_self`, but open to suggestions

Reply:
No better suggestion, the change is done